src/hooks/loginHooks/useSgidMultiuserLogin.ts

```diff
@@ -0,0 +1,24 @@
+import { AxiosError } from "axios"
+import { useMutation, UseMutationResult } from "react-query"
+
+import * as LoginService from "services/LoginService"
+
+import { VerifySgidMultiuserLoginParams } from "types/login"
+
+export const useSgidMultiuserLogin = (): UseMutationResult<
+  void,
+  AxiosError,
+  VerifySgidMultiuserLoginParams
+> => {
+  return useMutation<void, AxiosError, VerifySgidMultiuserLoginParams>(
+    (body) => LoginService.verifySgidMultiuserLogin(body),
+    {
+      onSuccess: () => {
+        window.location.replace("/sites")
+      },
+      onError: (err: AxiosError) => {
+        window.location.replace(`/ogp-login?status=${err.response?.status}`)
+      },
+    }
```

Comment:
`useHistory` here please. this is low prio so feel free to either ignore or do iin follow up

Reply:
See above!